scripts/revision-rapida.py

```
@@ -0,0 +1,628 @@
+    if n_errores == 0 and n_advertencias == 0:
+        lineas += [
+            "## Resumen",
+            "",
+            "✅ No se encontraron errores ni advertencias en el análisis estático.",
+            "",
+        ]
+    else:
+        lineas += [
+            "## Resumen",
+            "",
+            f"| Severidad | Cantidad |",
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)